presto-docs/src/main/sphinx/presto_cpp/features.rst

```diff
@@ -22,8 +22,8 @@ HTTP endpoints related to tasks are registered to Proxygen in
 
 Other HTTP endpoints include:
 
-* POST: v1/memory
-  * Reports memory, but no assignments are adjusted unlike in Java workers.
+* POST: v1/memory: Reports memory, but no assignments are adjusted unlike in Java workers
```

Comment:
I think this is not clear enough. Will you be able to add more explanations here?

Reply:
@karteekmurthys, could you respond to @yingsu00's comment?

Reply:
I have added more details now.